docs/notes/2.23.x.md

```diff
@@ -32,12 +32,12 @@ the nature of the changes in the source code that may lead to frequent cache inv
 
 ### update-build-files goal
 
-`buildifier` was added to the list of supported formatters that can be used to format the BUILD files. 
-It may be helpful if your organization is migrating from Bazel and wants to keep the style of the BUILD files 
-consistent or if for any other reason you may want to adopt the formatting style that is enforced by `buildifier`.  
+`buildifier` was added to the list of supported formatters that can be used to format the BUILD files.
```

Comment:
Sorry about these changes - looks like my editor took out some trailing whitespace on lines. If the whitespace is needed I'm happy to revert out the whitespace changes specifically.